apps/api/eslint.config.mjs

```diff
@@ -0,0 +1,10 @@
+import globals from "globals";
```

Comment:
Why is this file needed? Don't think we need it.

Reply:
Removed the file